docker-compose.yml

```diff
@@ -0,0 +1,12 @@
+version: '3'
+
+services:
+  randon_games:
+    build: .
+    container_name: "randon_games"
```

Comment:
randon games

Reply:
oh, sorry!

Reply:
Fixed the typo in commit [94978f7](https://github.com/Radon-Games/Radon-Games/pull/131/commits/94978f78a0c003a565557e518df60bb7e8daa16f)